src/config.ts

```diff
@@ -80,12 +80,11 @@ function validateLicenses(
   if (licenses === undefined) {
     return
   }
+
   const invalid_licenses = licenses.filter(license => !isSPDXValid(license))
 
   if (invalid_licenses.length > 0) {
-    throw new Error(
-      `Invalid license(s) in ${key}: ${invalid_licenses.join(', ')}`
```

Comment:
I removed the formatting since it was making it hard to debug failures. This resulted in a couple of changes in the tests where we perform string comparisons.